packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_controller.dart

```diff
@@ -121,6 +138,16 @@ class PropertyEditorController extends DisposableController
     );
   }
 
+  Timer _periodicallyCheckConnection(Duration interval) {
+    return Timer.periodic(interval, (timer) async {
+      final isClosed = await editorClient.isClientClosed();
+      if (isClosed) {
+        _shouldReconnect.value = true;
```

Comment:
does this value ever get set back to false?

Reply:
No, but because the IFRAME is then re-loaded the whole Property Editor is re-initialized again (with the default value `false`). I could reset to `false` though I don't know if it would ever actually be called.